docs/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -33,26 +33,22 @@ Configure Infinispan as a storage solution through the Caching service by settin
   This property specifies the list of cluster nodes (members). In case of multiple instances, the value for each Caching Service instance can be 
   either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.infinispan.jgroups.port}]`.
 
-* **`zowe.components.caching-service.storage.infinispan.persistence.dataLocation`**
-
-  The path where the service keeps its data files for the Infinispan Soft-Index Cache Store. 
-  The default value is `data`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the data property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
-
-
-* **`zowe.components.caching-service.storage.infinispan.persistence.indexLocation`**
-
-  The path where the service keeps its index data for the Infinispan Soft-Index Cache Store. 
-  The default value is `index`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the index property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
-
 
 * **`zowe.components.caching-service.storage.infinispan.jgroups.port`**
 
   (OPTIONAL)The default value is `7098`. The port number used by Infinispan to synchronise data among cahing-service instances.
 
+:::note
+We highly recommend to define this value even it is optional, because it is more resistance for future upgrading of Zowe.
```

Comment:
Does "resistance for future upgrade" mean? Could this be stated differently like, "We recommend you define this value to avoid potential problems (or errors) in future Zowe upgrades."?

Reply:
Yes, that sounds better, thanks.